pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -372,7 +433,9 @@ func (c *clusterAutoscaler) Destroy(ctx context.Context) error {
 		c.emptyVPA(),
 		c.emptyPodDisruptionBudget(),
 		c.emptyDeployment(),
-		c.emptyClusterRoleBinding(),
+		c.emptyClusterRoleBinding(), //TODO: @aaronfern Remove this after g/g:v1.119 is released
```

Comment:
```suggestion
		c.emptyClusterRoleBinding(),
```

Nit: Rather add the TODO to the function definition of `emptyClusterRoleBinding`. When applying the TODO statement there, you'll already get pointed to this usage which has to be removed.